### DISCUSSION TITLE
ComparisonError unable to find api field in struct Probe for the json field "grpc"

### DISCUSSION THREAD

**Discussion by MkrierPharmanity** [argoproj/argo-cd#12017] [Q&A]
Hello

when i deploy a simple project with a livenessProbe of type grpc i got this error

i'm deploying this application from this example : https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe

launching the kubectl apply from the terminal is working but with argocd it is not working

**Comment by MkrierPharmanity** [comment]
its seems that argocd is validating schema with a old version of kubernetes schema

if i use kubeval with this schema : https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.24.6-standalone-strict/deployment-apps-v1.json
i got no error 

it is possible to change where argocd is getting his schemas ?

**Reply by ps-jdollarhide** [reply]
Hey hey @MkrierPharmanity. We seem to be running into the exact same issue on our end. We are using a older version of ArgoCD than what is currently available, v2.0.4+0842d44 to be exact, and I would hope the issue might go away if we update versions, but were you ever able to solve this issue? Was there a way to update what schema ArgoCD validates against for the App Conditions?

**Comment by wv-rodney** [comment]
Bumping this discussion up. Would love some answers as well.
